docs/user-guide/horizontal-pod-autoscaling/index.md

```diff
@@ -22,19 +23,41 @@ to match the observed average CPU utilization to the target specified by user.
 
 ![Horizontal Pod Autoscaler diagram](/images/docs/horizontal-pod-autoscaler.svg)
 
-The autoscaler is implemented as a control loop.
-It periodically queries CPU utilization for the pods it targets.
+The autoscaler is implemented as a control loop
```

Comment:
"The Horizontal Pod Autoscaler is implemented as a control loop, with a period controlled by Controller Manager's `--horizontal-pod-autoscaler-sync-period` flag (with a default of 30 seconds). During each period, Controller Manager queries resource utilization against specific metrics. Controller Manager obtains these metrics from either the resource metrics API (for per-pod resource metrics) or the custom metrics API (for all other metrics)."